docs/README.md

```diff
@@ -2410,7 +2408,6 @@ Currently, the Booster framework provides a fully working provider package:
 
 Other providers packages are currently under development. Some of the features might be missing:
 
--  **framework-provider-local-\***. The Booster framework local provider combines in-memory databases with a GraphQL API served through a Node.js Express Server. The local runtime is a convenient and fast way to deploy and test your code in a local development environment. From the API and semantic perspectives, there are no differences from using a real cloud provider, it just runs locally!
 - **framework-provider-kubernetes-\***
 - **framework-provider-azure-\***
 
```

Comment:
Can you explain that Azure and Kubernetes are already implemented and not currently under development?

Reply:
But they are not fully working, right? 
So, should I say something like they're implemented but they're not fully working for now?

Reply:
Correct. We should explain we have experimental support on these providers